blocks/inspector-controls/text-control/index.js

```diff
@@ -9,12 +9,12 @@ import { withInstanceId } from '@wordpress/components';
 import BaseControl from './../base-control';
 import './style.scss';
 
-function TextControl( { label, value, help, instanceId, onChange, type = 'text', ...props } ) {
+function TextControl( { label, value, help, helpText, instanceId, onChange, type = 'text', ...props } ) {
```

Comment:
There's already a "help" prop, I think it's the same thing isn't it?

Reply:
As far as I can tell, the help prop is not visible, and there primarily for screen-readers. The explicit `helpText` I created actually outputs text that's visible below the textfield too.

Reply:
Should we make the other visible? Unless there's a specific reason for it being hidden, it seems like the same kind of value.

Reply:
Doh, how did I miss that, it looks like you're right. I'm going to take another stab at that.